test/contrib/bigquery_test.py

```diff
@@ -147,3 +153,26 @@ def output(self):
             }
         }
         run_job.assert_called_with('proj', expected_body, dataset=BQDataset('proj', 'ds', None))
+
+
+class BigQueryClientTest(unittest.TestCase):
+
+    def test_retry_succeeds_on_second_attempt(self):
+        client = MagicMock(spec=bigquery.BigQueryClient)
```

Comment:
I think we want to assert `client. _initialise_client` is invoked due to retry.

Reply:
👍